packages/tsconfig-reference/copy/pt/options/files.md

```diff
@@ -0,0 +1,26 @@
+---
+display: "Arquivos"
+oneline: "Incluir uma lista definida de arquivos, não suporta globs"
+---
```

Comment:
This is the docs for an individual tsconfig option: `"files"` - you can change the name here, but it will only change the header. 

At the top it will still say files:

![Screen Shot 2020-01-15 at 6 17 44 PM](https://user-images.githubusercontent.com/49038/72479787-6328f380-37c3-11ea-8bf4-1e08ae2ce72a.png)

Then further down it will show both this `display`, and the compiler flag name:
![Screen Shot 2020-01-15 at 6 18 21 PM](https://user-images.githubusercontent.com/49038/72479807-720fa600-37c3-11ea-85bc-7a91ce24d34f.png)

`oneline` is used in the playground TSConfig reference, you should have one for every option though.